ruby_event_store/spec/client_spec.rb

```diff
@@ -7,47 +7,47 @@ module RubyEventStore
     let(:client) { RubyEventStore::Client.new(repository: InMemoryRepository.new) }
     let(:stream) { SecureRandom.uuid }
 
-    specify 'publish returns :ok when success' do
-      expect(client.publish(TestEvent.new)).to eq(:ok)
+    specify 'publish returns self when success' do
+      expect(client.publish(TestEvent.new)).to eq(client)
```

Comment:
I usually tested it as `client.publish().publish()` but that's also fine :)

Reply:
hmmm that could be good places to start going with mutant annotations:

```
specify 'publish returns self when success', mutant: "RubyEventStore::Client#publish" do
```

which should help with making mutation testing run faster. Do you think you could try that here and in a few other places where you know which method you test.

Reply:
@paneq I'm not a fan (yet) of this `mutant: ` keyword, so I will pass on this here. Maybe another `good first issue` for that?